client/state/data-layer/wpcom/concierge/schedules/shifts/test/index.js

```diff
@@ -0,0 +1,65 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import {
+	conciergeShiftsFetchError,
+	fetchConciergeShifts,
+	storeFetchedConciergeShifts,
+	showConciergeShiftsFetchError,
+} from '../';
+import { updateConciergeShifts } from 'state/concierge/actions';
+import { CONCIERGE_SHIFTS_REQUEST } from 'state/action-types';
+
+// we are mocking impure-lodash here, so that conciergeShiftsFetchError() will contain the expected id in the tests
+jest.mock( 'lib/impure-lodash', () => ( {
+	uniqueId: () => 'mock-unique-id',
```

Comment:
nice job not having it return a `0` or a `1` - this is important because the default behavior will return a `1` and we can sometimes get false positives in our tests because we accidentally hid a bug in our mocks 🙃